client/src/components/main/chatwindow/CreateComment.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState} from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+/*
+Posts new comment to database when enter key pressed
+Set newComment to true using Parent's dispatch
+*/
+
+const jwt = '';
+const config = { headers: { authorization: `bearer ${jwt}` } };
+
+const CreateComment = props => {
+  const { setNewComment } = props;
+  const [comment, setComment] = useState('');
+
+  const handleSubmit = text => {
+    axios
+      .post('/api/comments', { user: '5e4deef29b70942e38591863', text: text }, config)
+      .catch(err => console.error(err));
+
+    setComment('');
+  };
+
+  const handleEnter = event => {
+    if (event.keyCode === 13) {
```

Comment:
What does this do?

Reply:
this checks whether the enter key was pressed by checking for the keyCode value of the enter key